site/content/docs/manifest/scheduled-job.md

```diff
@@ -0,0 +1,129 @@
+Currently, Copilot only supports the "Scheduled Job" type for tasks that are triggered either on a fixed schedule or periodically.
+
+<div class="separator"></div>
+
+<a id="image" href="#image" class="field">`image`</a> <span class="type">Map</span>  
+The image section contains parameters relating to the Docker build configuration.  
+
+<span class="parent-field">image.</span><a id="image-build" href="#image-build" class="field">`build`</a> <span class="type">String or Map</span>  
+If you specify a string, Copilot interprets it as the path to your Dockerfile. It will assume that the dirname of the string you specify should be the build context. The manifest:
+```yaml
+image:
+  build: path/to/dockerfile
+```
+will result in the following call to docker build: `$ docker build --file path/to/dockerfile path/to` 
+
+You can also specify build as a map:
+```yaml
+image:
+  build:
+    dockerfile: path/to/dockerfile
+    context: context/dir
+    args:
+      key: value
+```
+In this case, copilot will use the context directory you specified and convert the key-value pairs under args to --build-arg overrides. The equivalent docker build call will be: `$ docker build --file path/to/dockerfile --build-arg key=value context/dir`.
+
+You can omit fields and Copilot will do its best to understand what you mean. For example, if you specify `context` but not `dockerfile`, Copilot will run Docker in the context directory and assume that your Dockerfile is named "Dockerfile." If you specify `dockerfile` but no `context`, Copilot assumes you want to run Docker in the directory that contains `dockerfile`.
+ 
+All paths are relative to your workspace root. 
+
+<span class="parent-field">image.</span><a id="image-location" href="#image-location" class="field">`location`</a> <span class="type">String</span>  
+Instead of building a container from a Dockerfile, you can specify an existing image name. Mutually exclusive with [`image.build`](#image-build).    
+The `location` field follows the same definition as the [`image` parameter](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_image) in the Amazon ECS task definition.
+
+<div class="separator"></div>
+
+<a id="on" href="#on" class="field">`on`</a> <span class="type">Map</span>  
+The configuration for the event that triggers your job.
+
+<span class="parent-field">on.</span><a id="on-schedule" href="#on-schedule" class="field">`schedule`</a> <span class="type">String</span>  
+You can specify a rate to periodically trigger your job, supported entries:
+
+* @yearly
+* @monthly
+* @weekly
+* @daily
+* @hourly
+* @every {duration} (For example, "1m", "5m") 
+* rate({duration}) based on CloudWatch's [rate expressions](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#RateExpressions) 
+
+Alternatively, you can specify a cron schedule if you'd like to trigger the job at a specific time:  
+
```

Comment:
Should we also/first mention the non-AWS-flavored cron option?

Reply:
How do they declare those?

Reply:
Standard UNIX cron! 
https://en.wikipedia.org/wiki/Cron#Overview

They can just declare it like `schedule: "0 12 * * MON-FRI"`

Reply:
Done!